BEIMA.Client/src/pages/Users/AddUserPage.js

```diff
@@ -37,27 +39,46 @@ const AddUserPage = () => {
     let passwordErrors = {};
     
     if(userFields["Password"] === '') {
-      passwordErrors["Password"] = 'Pasword cannot be empty';
+      passwordErrors["Password"] = 'Password cannot be empty';
     }
 
     if(userFields["Password Confirmation"] === '') {
-      passwordErrors["Password Confirmation"] = 'Pasword Confirmation cannot be empty';
+      passwordErrors["Password Confirmation"] = 'Password Confirmation cannot be empty';
     }
     
     if((!('Password' in passwordErrors) && !('Password Confirmation' in passwordErrors)) &&
        (userFields["Password"] !== userFields["Password Confirmation"])
       ) {
-         passwordErrors["Password"] = 'Paswords do not match';
-         passwordErrors["Password Confirmation"] = 'Paswords do not match';
+         passwordErrors["Password"] = 'Passwords do not match';
+         passwordErrors["Password Confirmation"] = 'Passwords do not match';
     }
     
```

Comment:
Let's add in checks for an 8 character password length and require special/number/uppercase characters as well